config.py

```diff
@@ -83,6 +83,10 @@
 FNAME_BAD_EPOS_TEMPLATE = os.path.join(
     str(FPATH_DS), "derivatives", "EMP{sub:02}", "EMP{sub:02}_bad-epochs.json"
 )
+# Hypotheses tests
+FNAME_HYPOTHESES_2_TEMPLATE = os.path.join(str(FPATH_DS), "derivatives", "h2", "{h}")
+FNAME_HYPOTHESES_3_TEMPLATE = os.path.join(str(FPATH_DS), "derivatives", "h3", "{h}")
+FNAME_HYPOTHESES_4_TEMPLATE = os.path.join(str(FPATH_DS), "derivatives", "h4", "{h}")
 
```

Comment:
Can you follow the same pattern that I introduced for H1, see below?

`FNAME_REPORT_H1 = FPATH_DS / "derivatives" / "report-H1.html"`

i.e., I suggest we save the reports in the top level derivatives, not in "hypothesis" sub folders

Reply:
sure

Reply:
```python 
fname_report = Path(FNAME_REPORT_H2.format(h="h3_report.html"))
fname_h3a = Path(FNAME_HYPOTHESES_3_TEMPLATE.format(h="h3a_cluster.pkl"))
fname_h3b_wavelet = Path(FNAME_HYPOTHESES_3_TEMPLATE.format(h="h3b_wavelet.pkl"))
fname_h3b_cluster = Path(FNAME_HYPOTHESES_3_TEMPLATE.format(h="h3b_cluster.pkl"))
```

Reply:
I think saving TF results in hypothesis-wise folders is good :+1: I would just pull the "reports" to the top level.

Reply:
okay